### PR TITLE
[SDK Actions 1 of 2] DCOS-14910: Disable sdk service actions

### PR DESCRIPTION
**Pull request by mlunoe**
This PR introduces new modals for SDK services and applies them only to the services in the service table.

1 of 2: https://github.com/dcos/dcos-ui/pull/2144
2 of 2: https://github.com/dcos/dcos-ui/pull/2160

Test this by adding this repository for packages and installing the Hello Wolrd package:
https://infinity-artifacts.s3.amazonaws.com/autodelete7d/hello-world/20170501-114415-H2EXdmN1BlhErgpw/stub-universe-hello-world.zip

This PR merges to a feature branch and is not concerned with:
- group actions
- service detail actions
- pod detail actions
- service configuration
These will come in later PRs

(Preview doesn't seem to work 😢  Here is the link: https://cl.ly/kDyf/Screen%20Recording%202017-04-28%20at%2010.22.gif)

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [x] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

P.S. Don't be scared by the huge amount of additions. I took the liberty and created test for the service table with the same action tests that we already have for service detail.

### CODE REVIEW

File: plugins/services/src/js/constants/ServiceActionLabels.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const ServiceActionLabels = {
+  delete: "Delete",
+  restart: "Restart",
+  resume: "Resume",
+  scale: "Scale",
+  scale_group: "Scale By",
```
Comment (weblancaster) [inline plugins/services/src/js/constants/ServiceActionLabels.js]: I wonder if `scale_by` wouldn't make a bit more consistent, what you think? not a big deal though.

Comment (mlunoe) [inline plugins/services/src/js/constants/ServiceActionLabels.js]: Sure! Done!

**Comment by mlunoe** [comment]
Rebased

**Comment by mlunoe** [comment]
Run integration tests

**Comment by rcorral** [comment]
run integration tests

**Comment by rcorral** [comment]
Oops :)

**Comment by nLight** [comment]
Sorry, I have no idea how to test this. Don't know how to launch that magical job that has SDK label in it.

**Comment by rcorral** [comment]
@jfurrow could you post the command you've used?

**Comment by mlunoe** [comment]
Updated the description

File: src/js/translations/en-US.json [hunk 2]
```diff
@@ -8,5 +14,24 @@
   "DASHBOARD.PANEL_HEADING.COMPONENT_HEALTH": "Component Health",
   "DASHBOARD.PANEL_HEADING.NODES": "Nodes",
 
-  "SERVICE_ACTIONS.OPEN_SERVICE": "Open Service"
+  "DOCS.MORE_INFORMATION": "More information",
+  "DOCS.DOCUMENTATION": "documentation",
+
+  "SERVICE_ACTIONS.DELETE": "Delete",
```
Comment (weblancaster) [inline src/js/translations/en-US.json]: I believe we could use some of these in `COMMON` instead since it's very generic. Not big deal though.

**Pull request merged**